src/coreclr/tests/src/baseservices/exceptions/WindowsEventLog/WindowsEventLog.csproj

```diff
@@ -11,4 +11,7 @@
   <ItemGroup>
     <Compile Include="$(MSBuildProjectName).cs" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Is this supposed to be the  `TagList` ?   I actually like `TraitTags` more, just want to confirm this is intentional use of the name.

Reply:
Yes, _TraitTags_ is the name currently given to the taglist.

Reply:
I must admit I give a damn about the actual identifier. As you two guys are mostly working on this now and no work has been done to support these in the actual test projects AFAIK, I guess you two just figure out whatever naming you're happy with.

Reply:
Me too :-) . Just wanted to make sure this is not something different. 
`TraitTags` sounds good to me.